### PR TITLE
Added gremlin.graph setting to all config files.

### PR DESCRIPTION
**Pull request by MikeThomsen**
-----

Thank you for contributing to JanusGraph!

In order to streamline the review of the contribution we ask you
to ensure the following steps have been taken:

### For all changes:
- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
- [ ] Does your PR body contain #xyz where xyz is the issue number you are trying to resolve?
- [ ] Has your PR been rebased against the latest commit within the target branch (typically `master`)?
- [ ] Is your initial contribution a single, squashed commit?

### For code changes:
- [ ] Have you written and/or updated unit tests to verify your changes?
- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)?
- [ ] If applicable, have you updated the LICENSE.txt file, including the main LICENSE.txt file in the root of this repository?
- [ ] If applicable, have you updated the NOTICE.txt file, including the main NOTICE.txt file found in the root of this repository?

### For documentation related changes:
- [ ] Have you ensured that format looks appropriate for the output in which it is rendered?
- [ ] If this PR is a documentation-only change, have you added a `[skip ci]`
  tag to the first line of your commit message to avoid spending CPU cycles in
  Travis CI when no code, tests, or build configuration are modified?

### Note:
Please ensure that once the PR is submitted, you check Travis CI for build issues and submit an update to your PR as soon as possible.

### CODE REVIEW

**Comment by MikeThomsen** [comment]
This is intended to address [this issue](https://github.com/JanusGraph/janusgraph/issues/659) where the `gremlin.graph` setting had to be added to each connection property file tested against a backend.

**Comment by pluradj** [comment]
Thanks for the contribution @MikeThomsen!
The JanusGraph project uses a CLA and DCO for all contributions.
Please refer to the [CONTRIBUTING.md](https://github.com/JanusGraph/janusgraph/blob/master/CONTRIBUTING.md) on how to proceed.

File: janusgraph-dist/src/assembly/cfilter/conf/janusgraph-cassandra-configurationgraph.properties [hunk 1]
```diff
@@ -81,3 +81,4 @@ cache.db-cache-time = 180000
 # Data Type:  Double
 # Mutability: MASKABLE
 cache.db-cache-size = 0.5
+#JANUSGRAPHCFG{gremlin.graph=org.janusgraph.core.JanusGraphFactory}
```
Comment (chupman) [inline janusgraph-dist/src/assembly/cfilter/conf/janusgraph-cassandra-configurationgraph.properties]: This setting should not be added for properties files designed for configured graph factory.

File: janusgraph-dist/src/assembly/cfilter/conf/janusgraph-cql-configurationgraph.properties [hunk 2]
```diff
@@ -81,3 +81,4 @@ cache.db-cache-time = 180000
 # Data Type:  Double
 # Mutability: MASKABLE
 cache.db-cache-size = 0.5
+#JANUSGRAPHCFG{gremlin.graph=org.janusgraph.core.JanusGraphFactory}
```
Comment (chupman) [inline janusgraph-dist/src/assembly/cfilter/conf/janusgraph-cql-configurationgraph.properties]: This needs to be removed for this CFG properties file.

**Comment by FlorianHockmann** [comment]
@MikeThomsen: Do you plan to come back to this pull request and address the review comments, file a CLA and sign your commit (`git commit --amend -s`)?
Otherwise we unfortunately have to close it.

**Comment by chupman** [comment]
Closing due to inactivity.

**Pull request closed without merging**